build/love/manager-static/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Clone repo
+FROM alpine/git as cloner
+
+WORKDIR /usr/src/love
+ARG love_manager
+RUN git clone --branch ${love_manager} --depth 1 --single-branch https://github.com/lsst-ts/LOVE-manager.git
+
+FROM python:3.8.2-buster as builder
+COPY --from=cloner /usr/src/love/LOVE-manager /usr/src/love/LOVE-manager
```

Comment:
This is a general comment that applies to all of the directories specified like this in both manager-static and manager Dockerfiles. This seems overly redundant. Is there something in the package reorg that necessitated changing this?

Reply:
You mean using a container to clone and them copy it over? If yes, then the problem is that `python:3.8.2-buster` does not contain git, so I can't really clone the repo. I thought using this construct was better than installing git.

Reply:
No. I meant the `/usr/src/love/LOVE-manager`. Seems like `/usr/src/love/manager` is much simpler.